ci-operator/config/openshift/windows-machine-config-operator/openshift-windows-machine-config-operator-master.yaml

```diff
@@ -146,6 +146,17 @@ tests:
     test:
     - ref: windows-e2e-operator-test-byoh
     workflow: windows-e2e-operator-test-platform-none-vsphere
+- as: wicd-unit-vsphere
+  steps:
+    cluster_profile: vsphere-platform-none
+    post:
+    - chain: upi-vsphere-windows-post
+    pre:
+    - ref: ipi-install-rbac
+    - chain: upi-conf-vsphere-platform-none-ovn
+    - chain: upi-vsphere-windows-pre
```

Comment:
I feel it should be beneficial to create an explicit workflow just with above `pre` and `post` chains, where this test may read:

```yaml
- as: wicd-unit-vsphere
  steps:
    cluster_profile: vsphere-platform-none
    test:
    - ref: windows-unit-wicd
    workflow: upi-vsphere-windows-install
```

Also, it can be reused for other tests.

Reply:
i'd prefer to hold off on creating a workflow for this until there is another case where we need a windows VM without a cluster
this seems like a one-off, and I'd like to not clutter the step-registry further, without good reason of course